cmd/frontend/backend/repos_test.go

```diff
@@ -235,8 +235,6 @@ func TestReposGetInventory(t *testing.T) {
 
 func TestMain(m *testing.M) {
 	flag.Parse()
-	if !testing.Verbose() {
-		log15.Root().SetHandler(log15.DiscardHandler())
-	}
+	logtest.Init(m)
```

Comment:
We should preserve the silencing behaviour here:

```suggestion
	if !testing.Verbose() {
		logtest.InitWithlevel(m, log.LevelNone)
	} else {
		logtest.Init(m)
	}
```